### DISCUSSION TITLE
RocketMQ 5.x 是否有完善的文档？

### DISCUSSION THREAD

**Discussion by zergduan** [apache/rocketmq#8646] [General]
RocketMQ 5.x 架构发生改变增加了 Proxy 组件，但是对于这个组件的文档我没有找到，请问：

1. 是否有完善的文档描述新架构应该如何配置？例如：
如何给Proxy设置JVM内存？
Proxy参数rmq-proxy.json总可以配置哪些参数？
Proxy应该如何设置TLS？（我们需要配置Producer和Consumer与Proxy保持TLS连接，而其它组件之间保持非TLS连接）
以前启动NameServer和Broker的方式是执行 runserver.sh 和 runbroker.sh 脚本，现在应该如何启动才能带上正确的JVM环境变量和参数？
2. 如果没有完善的文档，是否能提供放弃使用Proxy的部署方式，保持与4.x一样：只有Nameserver和Broker组件。

**Comment by zergduan** [comment]
另外Proxy如何设置ACL？不同用户对不同topic有不同权限，原来的架构这个设置在broker中，现在client只连接proxy，理论上应该在proxy上完成鉴权，请问这个proxy如何开启ACL，如何配置ACL权限？

**Reply by YongXiaoo** [reply]
可以查看ProxyConfig.java文件来看，proxy支持ACL
